components/OurRecruiter.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from 'react';
+import Image from 'next/image';
+
+import image3 from '@assets/banner/ankur.jpg';
+import image2 from '@assets/banner/ashpreet.jpg';
+import image1 from '@assets/banner/ourrecuiters.jpg';
+
+const OurRecruiter: React.FC = () => {
```

Comment:
pls. change file name too: `OurRecruiter.tsx -> OurRecruiters.tsx`

```suggestion
const OurRecruiters: React.FC = () => {
```